app/src/components/widgets/charts/passingRatePerLaunch/passingRatePerLaunch.jsx

```diff
@@ -10,6 +10,7 @@ import { redirect } from 'redux-first-router';
 import { defectLinkSelector, statisticsLinkSelector } from 'controllers/testItem';
 import * as COLORS from 'common/constants/colors';
 import { STATS_PASSED } from 'common/constants/statistics';
+import { CHART_MODES } from 'pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/constants';
```

Comment:
Looks strange that widget depends on something from creation modal. I think it's better to move this constant to an other place.

Reply:
Done